blocks/bricks/wrappers.py

```diff
@@ -34,3 +35,35 @@ def apply(self, input_):
         else:
             output = self.application_method(input_)
         return output
+
+
+class WithAxesSwapped(Brick):
+    """Wraps an application around dimshuffles.
+
+    Parameters
+    ----------
+    application_method : callable
+        Some brick's :meth:`apply` method
+    dim0 : int
+        First dimension to swap
```

Comment:
Looks slightly better in the rendered docs (and more consistent with the rest of the documentation) to terminate these descriptions with a period.